### PR TITLE
Refractor health

### PR DESCRIPTION
**Pull request by mpikzink**
Reduce equal code which makes it easier in the future to add new sensors.

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Review by Jellyfrog — APPROVED** [review]

**Comment by CirnoT** [comment]
Note, this breaks Health tab on UI. All tabs are the same and stuff like Disk I/O does not work anymore as it attempts to load `device_ucd_diskio` instead of `device_diskio`.

chrome_2021-07-11_14-03-48

In addition clicking tabs loads URL via `/device/device=41/tab=health/metric=ucd_diskio/` which does nothing. Manually changing to `/device/device=41/tab=health/metric=diskio/` fixes it and shows correct graphs.

File: includes/html/pages/device/health.inc.php [hunk 1]
```diff
@@ -16,252 +13,89 @@
...
-$type_text['delay'] = 'Delay';
-$type_text['quality_factor'] = 'Quality factor';
-$type_text['chromatic_dispersion'] = 'Chromatic Dispersion';
-$type_text['ber'] = 'Bit Error Rate';
-$type_text['eer'] = 'Energy Efficiency Ratio';
-$type_text['waterflow'] = 'Water Flow Rate';
-$type_text['loss'] = 'Loss';
-$type_text['qfp'] = 'QFP';
-$type_text['percent'] = 'Percent';
+    foreach (Sensor::getTypes() as $sensor) {
```
Comment (murrant) [inline includes/html/pages/device/health.inc.php:46]: This was kinda silly, should have been `DeviceCache::getPrimary()->sensors()->select('sensor_class')->selectSub('COUNT(*)', 'sensor_count')->groupBy('sensor_class')->pluck('sensor_count', 'sensor_class');`

```php
[
       "fanspeed" => 1,
       "power" => 1,
       "state" => 14,
       "temperature" => 3,
       "voltage" => 3,
     ]
```

**Comment by mpikzink** [comment]
Thanks @CirnoT for the information it results in different sensor names with some sensor meters. @Jellyfrog already revert the PR. And thanks @murrant  this seems to be a very clean way so select the sensors. I will try a new way... Sorry!

**Comment by murrant** [comment]
@mpikzink Yeah, I was looking at the old code and wondering we had a billion queries, then I looked at the new code and it was the same, just in a tidy loop :)  No worries, thanks for your contribution.

**Pull request merged**